modules/heat_conduction/test/tests/thin_layer_heat_transfer/tests

```diff
@@ -0,0 +1,42 @@
+[Tests]
+  [steady_2d]
+    type = 'Exodiff'
+    input = 'steady_2d.i'
+    exodiff = 'steady_2d_out.e'
+    requirement = 'The system shall model steady state heat transfer with an interface between two domains in 2D.'
+    issues = '#21988'
+    design = 'ThinLayerHeatTransfer.md'
+    mesh_mode = 'REPLICATED'
+    recover = false
```

Comment:
these restrictions are because of the `BreakMeshByBlockGenerator`, right?

Reply:
That is correct.